AngularJS/js/controllers/dashboardController.js

```diff
@@ -0,0 +1,88 @@
+var cont=app.controller('dashboardController', ['$scope', function($scope) {
+  $scope.query = {}
+  $scope.queryBy = '$'
+  $scope.books = [
```

Comment:
Don't create `$scope` variables. Those would be global variables and we consider them not the best practice and highly avoidable. Use `this` to add variables to the controller. In this case, it would be:

```js
this.query = {};
this.queryBy = '';
this.books = [...];
```